assets/app/styles/_core.less

```diff
@@ -110,15 +113,20 @@ html, body {
   }
 }
 @media (min-width: @screen-sm-min) {
-  .console-os .navbar-project {
-    .navbar-filter-widget {
-      /* Prevent the filter button from wrapping at wide screen sizes. */
-      min-width: 340px;
+  .console-os {
+    .navbar-project {
+      .navbar-filter-widget {
+        /* Prevent the filter button from wrapping at wide screen sizes. */
+        min-width: 340px;
+      }
+      .create-button-container {
+        text-align: right;
+        /* Align button with other navbar controls that have a label above. */
+        margin-top: 30px;
+      }
     }
-    .create-button-container {
-      text-align: right;
-      /* Align button with other navbar controls that have a label above. */
-      margin-top: 30px;
+    .container-main {
+      min-height: 600px;
```

Comment:
Yes, this is to prevent a bug in IE 10/11. 

This stems from not setting .wrap to height: 100%,* thus on IE, pages with short content have a bug where the sidebars don't extend to the bottom of the viewport. Ideally I would have targeted ie only to set height, but we're not using modernizr to generate the ie class on html tag. So I'm giving container-main a min-height so that at least it's taller than the left nav.

* bug on chrome related to height:100% https://bugzilla.redhat.com/show_bug.cgi?id=1215023